spring-boot-project/spring-boot/src/main/java/org/springframework/boot/web/servlet/ServletRegistration.java

```diff
@@ -87,4 +90,50 @@
 	 */
 	int loadOnStartup() default -1;
 
+	/**
+	 * Init parameters to set on the servlet, as {@code "key=value"} pairs.
+	 */
+	String[] initParameters() default {};
+
+	/**
+	 * (Optional) Additional servlet-registration beans to apply.
+	 * Usually left empty unless you need custom bean logic.
+	 */
+	Class<? extends ServletRegistrationBean<?>>[] servletRegistrationBeans() default {};
+
+	/**
+	 * Multipart configuration. Mirrors {@link jakarta.servlet.annotation.MultipartConfig}.
+	 * If you omit it (no fields changed), it will not set a multipart config.
+	 */
+	MultipartConfigValues multipartConfig() default @MultipartConfigValues;
+
+	/**
+	 * Nested annotation that parallels the fields of
+	 * {@link jakarta.servlet.annotation.MultipartConfig}. Used within
+	 * {@link ServletRegistration#multipartConfig()}.
+	 * @see jakarta.servlet.annotation.MultipartConfig
+	 */
+	@Target({})
+	@Retention(RetentionPolicy.RUNTIME)
+	@Documented
+	@interface MultipartConfigValues {
```

Comment:
Could we directly use `jakarta.servlet.annotation.MultipartConfig` here?

Reply:
yes, good catch

Reply:
done